doc/api/errors.md

```diff
@@ -1831,6 +1831,11 @@ unknown file extension.
 
 Creation of a [`zlib`][] object failed due to incorrect configuration.
 
+### MODULE_NOT_FOUND
```

Comment:
1. Should not this be `### ERR_MODULE_NOT_FOUND` and be placed in a sorted way?
2. It seems this doc does need `<a id=...` fragments.

Reply:
I believe the actual error name is currently just `MODULE_NOT_FOUND`.

Reply:
So this is the first error name out of the common name pattern?